File: woodroid/src/main/java/me/gilo/woodroid/data/RestAdapter.java
package me.gilo.woodroid.data;

import org.apache.http.NameValuePair;

import java.util.ArrayList;

public class RestAdapter {

    static String oauth_nonce = "";
    static String oauth_timestamp = "";
    static String oauth_signature_method = "HMAC-SHA1";

    static ArrayList<NameValuePair> params;

    private String baseUrl;
    private String consumerKey;
    private String consumerSecret;

    public RestAdapter(String baseUrl, String consumerKey, String consumerSecret) {
        this.baseUrl = baseUrl;
        this.consumerKey = consumerKey;
        this.consumerSecret = consumerSecret;
    }




}

File: cocart/src/main/java/me/gilo/cocart/data/RestAdapter.kt
package me.gilo.woodroid.data

import org.apache.http.NameValuePair

import java.util.ArrayList

class RestAdapter(private val baseUrl: String, private val consumerKey: String, private val consumerSecret: String) {
    companion object {

        internal var oauth_nonce = ""
        internal var oauth_timestamp = ""
        internal var oauth_signature_method = "HMAC-SHA1"

        internal var params: ArrayList<NameValuePair>? = null
    }


}
